### ISSUE TITLE
Not keeping track of validation set performance during training?

### ISSUE THREAD

**Issue by thomasjanssens** [yhenon/keras-frcnn#84]
Is there a reason the current version of the training script does not appear to keep track of validation set performance during training? It seems `data_gen_val` is defined but no longer used, is this an oversight perhaps?

**Comment by shahruk10** [comment]
I believe it is used when measuring the mean average precision (mAP)

**Comment by maxrichter-scdm** [comment]
I was asking this myself - so far I did not find out where it is used

**Comment by yhenon** [comment]
This is an oversight on my part and should be added as an option (since it can be kind of time consuming and the original implementation didnt have it)

**Comment by apacha** [comment]
Related issues: #172, #165
